storm-client/test/jvm/org/apache/storm/utils/ConfigUtilsTest.java

```diff
@@ -92,32 +92,32 @@ public void getValueAsList_nonStringList() {
     }
 
     @Test
-    public void getBlobstoreHDFSPrincipal() throws UnknownHostException {
```

Comment:
do we want to keep this test as is for validating deprecated configs?  It looked like they should still work?

Reply:
Makes sense. Will fix it.